### PR TITLE
add com.github.vikdevelop.timer.yml

### PR DESCRIPTION
**Pull request by vikdevelop**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer.yml

File: com.github.vikdevelop.timer.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+       - install -Dm755 -t /app/share/applications flatpak/com.github.vikdevelop.timer.desktop
+       - install -Dm755 -t /app/share/icons/hicolor/128x128/apps flatpak/icons/com.github.vikdevelop.timer.png
+       - install -Dm644 -t /app/share/metainfo flatpak/com.github.vikdevelop.timer.metainfo.xml
+    sources:
+          - type: archive
+            url: https://github.com/vikdevelop/timer/archive/refs/tags/1.3.tar.gz
+            sha256: 0b6172c82e813222cc6b74045851850376f0f8ba9568eb0e3d6efb4b61f8d5d3
+          - type: archive
+            url: https://download.gnome.org/sources/pygtk/2.24/pygtk-2.24.0.tar.gz
+            sha256: 6e3e54fa6e65a69ac60bd58cb2e60a57f3346ac52efe995f3d10b6c38c972fd8
```
Comment (hfiguiere) [inline com.github.vikdevelop.timer.yml:26]: you want this in a new module. this won't work otherwise.

**Comment by hfiguiere** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Comment by vikdevelop** [comment]
bot, build com.github.vikdevelop.timer

**Pull request closed without merging**